frontend/resourceadm/components/MigrationPanel/MigrationPanel.tsx

```diff
@@ -67,6 +70,44 @@ export const MigrationPanel = ({
     setServiceExpiredWarningModalRef.current?.close();
   };
 
+  const getErrorMessage = (): React.ReactNode => {
```

Comment:
Could maybe also change this line to this? 
```suggestion
  const ErrorMessage = (): ReactElement => {
```